src/utils/validations/component/validations/validations.context.js

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export default React.createContext();
```

Comment:
Seems like this could be a one-liner in whatever files you're importing this into?